setup.py

```diff
@@ -90,3 +91,22 @@ def long_description():
         'mypy_extensions>=0.4',
     ],
 )
+
+
+print(in_color('green', '\nZulip Terminal was successfully installed!'))
+if MACOS:
+    sys.exit(0)
+print('\nTo ensure notifications are displayed, you may wish to run the '
+      'following:\n')
+if WSL:
+    print(in_color('blue', 'Run powershell as "admin" and run these commands'
+                   ' to install dependencies for showing notifications:'))
+    print(in_color('red', '\nset-executionpolicy remotesigned\nInstall-Module'
+                          ' -Name BurntToast\n'))
+elif LINUX:
+    print(in_color('blue', 'The following command installs `notify-send` on'
+                   ' Debain based systems, similar commands can be found for'
+                   ' other linux systems as well.'))
+    print(in_color('red', '\nsudo apt-get install libnotify-bin\n'))
+print('You can refer to https://github.com/zulip/zulip-terminal#notifications'
+      ' for more info on showing notifications.')
```

Comment:
Did you try installing the package and testing this?

`python setup.py install` does show these messages, but `pip install .` does not.

Reply:
Looks very tricky to get it working for pip install . I only checked for `python setup.py install ` prior to this. I tried a few things but nothing is working. Any suggestions?

Reply:
I did briefly research this before, but those solutions also don't appear to work with setup.py. I think this comes down to using wheels. You do get the output if you use '-vvv', but we can't expect that.

Can we integrate this element into the onboarding PR instead?

Reply:
Yeah, that sounds like a better idea. I will include it in the onboarding PR.